internal/ingress/controller/nginx.go

```diff
@@ -202,8 +202,9 @@ func NewNGINXController(config *Configuration, mc metric.Collector) *NGINXContro
 	}
 
 	for _, f := range filesToWatch {
+		f := f
```

Comment:
Based on the fact that it took me some time to understanding the root cause and the fix, I'd recommend adding a few lines of docs here. That's probably an easy one for devs deeper into that project and Go in general, but others might drop or refactor that line without further context. Maybe also reference this PR in your comment. Thank you!

Reply:
Added a link to a doc in the PR description.

Reply:
Sure, but nobody will find that if they are not `git blame`ing and walking through history. So please add some words next to your code addition. 🙂

Reply:
Added a description of the code to the code.